apps/web/src/store/app/datacenter/index.tsx

```diff
@@ -130,9 +125,6 @@ export function DataCenterPreloader({ children }: React.PropsWithChildren) {
         dataCenterPageList: [],
       });
     });
-    throw promise;
-  }
-  if (!dataCenter) {
```

Comment:
I think we should either use a third-party library or make our own abstraction to make the suspense pattern more clear and maintainable.